src/DemoData.hs

```diff
@@ -156,6 +187,9 @@ universe rnd = void $ do
 
 -- * Helpers
 
+-- This 'Monad m => m a' is strange, why not simply 'a' instead?
+-- Second, is this actually safe to use the same generator over and over or
+-- does it need to be threaded?
```

Comment:
There is one `GenT` on hackage: https://hackage.haskell.org/package/QuickCheck-GenT